### DISCUSSION TITLE
advection diffusion simulation questions

### DISCUSSION THREAD

**Discussion by oscarhx** [PMEAL/OpenPNM#2344] [Q&A]
Hello Professors,
I recently read the advection diffusion module. This is something I never see on public prints. There are some problems confused me for a long time.
1. the numerical solution of advection diffusion equation, like the upwind method, is derived from even grid using difference algorithm, so can the ad_dif_conductance.py be applied to random pore network structure, like Voronoi graph? 
2. the peclet number is defined as the ratio of velocity to the (diffusivity/character length). In the ad_dif_conductance.py is defined as: flux/diffusivity conductivity. How to testify such conversion, or how is the numerical solution of advection diffusion adapt to the pore network system?
Best,
oscarhx

**Comment by jgostick** [comment]
Hi @oscarhx , have you also read our [paper on the subject](https://doi.org/10.1016/j.jconhyd.2019.103578)?  This outlines the discretizations and provides many tests, validations and comparisons.
